src/Console/Command/ProcessCommand.php

```diff
@@ -157,6 +163,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->processCommandReporter->reportFileDiffs($this->fileDiffs);
         $this->processCommandReporter->reportChangedFiles($this->changedFiles);
 
+        if ($this->errors) {
+            $this->processCommandReporter->reportErrors($this->errors);
+            return 1;
```

Comment:
Instead of this we usually recommend using a speaking const, in cake core we have here

    public const CODE_ERROR = 1;

to make it clear why 1 is here not a succesful one.

Reply:
I had it somewhere but dropped after while.
Good idea. Where do you place it in application with many commands?

Reply:
Usually the base command, to use static::XYZ
I am surprised the framework doesnt offer this, as they should.

Symfony, CakePHP usually all do.

Reply:
Ref https://github.com/cakephp/cakephp/blob/master/src/Console/Shell.php